tools/wptserve/wptserve/request.py

```diff
@@ -319,6 +318,7 @@ def cookies(self):
         if self._cookies is None:
             parser = BaseCookie()
             cookie_headers = self.headers.get("cookie", b"")
+            # FIXME: cookies should probably be decoded in UTF-8?
```

Comment:
I'm not sure about this. I'm now leaning more towards storing cookies as bytes.

Reply:
Remove this comment for now then.

Reply:
```suggestion
```